<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>583</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>기부금영수증 발급 현황</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>92</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>발급년도:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="target_year_widget">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>117</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>142</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="cont_view_tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>156</y>
     <width>711</width>
     <height>351</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>발급일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>발급기호</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>대상년도</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>신청자</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>합산코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>확인금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>발행구분</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="view_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>보기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="end_close_Button">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>119</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>발행금액 합계</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>553</x>
     <y>124</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>재발행</string>
   </property>
  </widget>
  <widget class="QLabel" name="re_issue_count_widget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>122</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>126</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>건</string>
   </property>
  </widget>
  <widget class="QLabel" name="issued_amount_widget">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>116</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>104</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>최초발행</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>643</x>
     <y>103</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>건</string>
   </property>
  </widget>
  <widget class="QLabel" name="first_issue_count_widget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>101</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 발행금액 합계는 재발행 금액은 포함하지 않습니다.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="issue_year_widget">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>119</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>발행대상년도:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>93</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_print_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>프린트</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>issue_year_widget</tabstop>
  <tabstop>target_year_widget</tabstop>
  <tabstop>view_Button</tabstop>
  <tabstop>end_close_Button</tabstop>
  <tabstop>cont_view_tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>view_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>serch_view()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>end_close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>end_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_print_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>contribution_list_print()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>serch_view()</slot>
  <slot>end_close()</slot>
  <slot>contribution_list_print()</slot>
 </slots>
</ui>
